.github/workflows/java-client-release-to-maven.yml

```diff
@@ -27,7 +27,6 @@ jobs:
 
     - name: Build
       run: |
-        cp client/java/settings.xml /home/runner/.m2
         cd client/java
         mvn clean install
       env:
```

Comment:
This env is not needed for `install` target, the artifacts are signed during `deploy`.

Reply:
Sorry i don't get what you mean?